weblate/utils/validators.py

```diff
@@ -61,7 +61,7 @@
     '.rtf',
     '.xls',
     '.docx',
-    '.xlsx',
+    #'.xlsx',   #allowed due to xlsx translation upload
```

Comment:
Please remove the line instead of commenting it out.

Reply:
sure, I will do